prereise/cli/helpers.py

```diff
@@ -0,0 +1,45 @@
+import os
+from datetime import datetime
+
+
+def validate_date(date_string):
+    """Helper function to validate date strings
+
+    :param str date_string: date in string format to be validated
+    :return: (*str*) validated date string
+    """
+    datetime.strptime(date_string, "%Y-%m-%d")
+    return date_string
+
+
+def validate_file_path(file_path):
+    """Helper function to validate file paths
+
+    :param str file_path: path include filename for where
+        to save the file
+    :raises ValueError: if the provided file path is not accessible
+    :return: (*str*) validated file path
+    """
+    if not os.access(os.path.dirname(file_path), os.W_OK):
+        raise ValueError("Please choose a valid file path")
```

Comment:
We can also make sure that `file_path` is not a directory (fails with an `IsADirectoryError` after all the data has been downloaded). It might also be helpful to have a warning or confirmation prompt if the file already exists and will be overwritten.

Reply:
will do, added additional tests as well